### DISCUSSION TITLE
FR: Make it possible to not capture scrolling

### DISCUSSION THREAD

**Discussion by kimbostroem** [noah-nuebling/mac-mouse-fix#636] [Other]
Some programs (e.g. JASP data sheet) do not work properly with the scrolling controlled by MouseFix. It is already possible to release the control of mouse buttons by selecting all captured feature of that button (e.g. of middle button) and pressing the minus button, it should also be possible to release the control of the mouse wheel, e.g. by deselecting a checkbox.

**Comment by NeverRIEght** [comment]
Same suggestion.
For example in Minecraft MMF scrolling is scrolling throw all of my inventory instead of switching to the next slot.
And it is just one example, it is just getting worse in other apps.

It will be so kind of you to let us control scrolling. Or just add linear scrolling feature, which also will be fine.
